alerts/alerts.yml

```diff
@@ -0,0 +1,83 @@
+groups:
+  - name: Depositor bot. No metrics
+    rules:
+      - alert: DepositorBotNoMetrics
+        expr: absent_over_time(depositor_bot_build_info{service="lido-depositor-bot"}[5m])
+        for: 3m
+        labels:
+          severity: warning
+          app_team: tooling
+        annotations:
+          summary: "Depositor doesn't reporting metrics for too long."
+          description: "Depositor doesn't reporting metrics for too long."
+  - name: Pauser bot. No metrics
+    rules:
+      - alert: PauserBotNoMetrics
+        expr: absent_over_time(pauser_bot_build_info{service="lido-pause-bot"}[5m])
+        for: 3m
+        labels:
+          severity: warning
+          app_team: tooling
+        annotations:
+          summary: "Pauser doesn't reporting metrics for too long."
+          description: "Pauser doesn't reporting metrics for too long."
+  - name: Unvetter bot. No metrics
+    rules:
+      - alert: UnvetterBotNoMetrics
+        expr: absent_over_time(unvetter_bot_build_info{service="lido-unvetter-bot"}[5m])
+        for: 3m
+        labels:
+          severity: warning
+          app_team: tooling
+        annotations:
+          summary: "Unvetter doesn't reporting metrics for too long."
+          description: "Unvetter doesn't reporting metrics for too long."
+  - name: Transport stuck
+    rules:
+      - alert: NoMessagesReceivedFor6h
+        expr: sum by (service,address) (changes(label_replace({__name__=~"depositor_bot_ping_messages|depositor_bot_deposit_messages|pauser_bot_ping_messages|pauser_bot_pause_messages|unvetter_bot_ping_messages|unvetter_bot_unvet_messages"},"name_label","$1","__name__", "(.+)")[6h:])) == 0
```

Comment:
Do not aggregate by `address`. This should be a common event if bot can't receive messages.